static/app/views/onboarding/targetedOnboarding/setupDocs.tsx

```diff
@@ -149,6 +163,8 @@ function SetupDocs({organization, projects, search}: Props) {
     <React.Fragment>
       <Wrapper>
         <TargetedOnboardingSidebar
+          projects={projects}
+          selectedProjects={clientState.platforms || {}}
```

Comment:
@Neo-Zhixing It seems odd to map the client state platforms to `selectedProjects`. I'd expect `selectedProjects` to be some sort of array instead of a mapping. Could we get a different variable name?

Reply:
sure. how about "onboardingPlatformState"